### PR TITLE
Add ring tests

### PR DESCRIPTION
**Pull request by arufanov**
Add ring tests

### CODE REVIEW

**Comment by arufanov** [comment]
@yurichervoniy 
* lets discuss entity and dto tests.

File: src/test/java/com/a5lab/tabr/domain/rings/RingTests.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+
+class RingTests extends AbstractRepositoryTests {
+
+  @Autowired
+  private Validator validator;
+
+  @Test
+  void ensureValidatorIsLoaded() {
+    // https://stackoverflow.com/questions/29069956/how-to-test-validation-annotations-of-a-class-using-junit
```
Comment (yurichervoniy) [inline src/test/java/com/a5lab/tabr/domain/rings/RingTests.java:21]: You are not writing unit tests
AbstractRepositoryTests - causes spring context to load

**Comment by arufanov** [comment]
@yurichervoniy 
* could you provide example how test RingDto validation?

File: src/test/java/com/a5lab/tabr/domain/rings/RingTests.java (hunk 1 above)
Comment (arufanov) [reply src/test/java/com/a5lab/tabr/domain/rings/RingTests.java]: @yurichervoniy 
* fixed

**Comment by arufanov** [comment]
@yurichervoniy 
* could you review this test?
* I have only one questions: why I am not able to see this test at jacoco? I wish to see annotation testings.

**Pull request merged**